apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -703,7 +703,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
             } else {
                 metadataProp.options.map((ele: any, index: number) => {
                     if (!ele.displayName) {
-                        allowedList.push({ text: ele, value: ele });
+                        allowedList.push({ text: ele, value: ele !== "Select Option" ? ele : "" });
```

Comment:
Shall we add `i18n` placeholders for ` "Select Option"`

Reply:
Added with 00b0dabc60cdfb5f03ee413d60f0206d344e35ba